browser/src/control/jsdialog/Widget.DrawingArea.js

```diff
@@ -43,8 +43,9 @@ function _drawingAreaControl (parentContainer, data, builder) {
 	image.ondragstart = function() { return false; };
 	builder.map.uiManager.enableTooltip(image);
 
-	// If the image render is delayed, use width and height of the data
-	if (image.width == 0 && image.height == 0) {
```

Comment:
I don't understand why it doesn't work in line width case?
Maybe we should use OR not AND in the condition?

Reply:
It would be good to put more details into commit message, what exactly happens here

Reply:
That is a regression of https://github.com/CollaboraOnline/online/pull/7744

The issue is described here: https://github.com/CollaboraOnline/online/pull/7744

Actually it works on line width case but doesn't work on spellcheck dialog. Both dialogs are jsdialog have  drawing areas. Both image in the dialog delays at first time. Width and height is 0 at first time. And for the problematic spellcheck dialog data.image width and height is huge. So we see the following result:

![Screenshot from 2024-01-10 15-10-11](https://github.com/CollaboraOnline/online/assets/2954981/4fecc00a-be88-4d14-9f6e-e6801a5a9968)

Difference is line width dialog is relocating after then created. But spell check dialog is fixed we don't change the spellcheck dialog's position. Most probably that is the difference between two dialog. And couln't find a better way than id to differenciate them

Reply:
Ok. So please always reference in the commit message previous commit which caused regression if you know about that. It's much easier to review and later if someone will have to work on that code again.

I see that when I sometimes click in spellchecking dialog, I got new jsdialog update and it is correctly shown.
Sometimes it adds width and height, sometimes not. Would be good to know why.